test/unit/puppetlabs/services/jruby/jruby_testutils.clj

```diff
@@ -20,12 +17,6 @@
 (def gem-home "./target/jruby-gem-home")
```

Comment:
Seems like we should be able to nuke the puppet-specific `ruby-load-path` from these tests.